### PR TITLE
FOR REVIEW ONLY - go-algorand-sdk v1.10.0

### PR DESCRIPTION
**Pull request by algobarb**
# 1.10.0
# Added
- New github Issue template
- Signing support for rekeying to LogicSig/MultiSig account
- Asset Base64 Fields
# BugFix
- Use correct go version in CI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# 1.10.0
+# Added
+- New github Issue template
+- Signing support for rekeying to LogicSig/MultiSig account
```
Comment (jasonpaulos) [inline CHANGELOG.md:4]: This is missing https://github.com/algorand/go-algorand-sdk/pull/233, which adds the new base64 asset fields

**Review by jasonpaulos — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (algobarb) [reply CHANGELOG.md:4]: Yup, I'm adding it now.

Comment (algobarb) [reply CHANGELOG.md:4]: #233 is added in Changelog and Release draft now.

**Review by jasonpaulos — APPROVED** [review]
Looks good

**Review by onetechnical — APPROVED** [review]

**Pull request merged**